apps/server/test/lexical/server/code_intelligence/completion_test.exs

```diff
@@ -52,20 +52,16 @@ defmodule Lexical.Server.CodeIntelligence.CompletionTest do
       end
     end
 
-    test "ignores erlang modules", %{project: project} do
-      assert %Completion.List{is_incomplete: true, items: []} =
-               complete(project, ":e|", as_list: false)
+    test "completes erlang modules", %{project: project} do
+      assert [_ | _] = completions = complete(project, ":e|")
```

Comment:
I did this intentionally; having a single character completion for basically any atom creates a _lot_ of noise when writing elixir code. Completing an erlang module is a rare-ish event compared to using atoms. I know that this change is to improve things for vscode's broken behavior, but i don't like the ux.